src/locales/fr/UI/IMPORT.json

```diff
@@ -0,0 +1,17 @@
+{
+    "TITLE": "Importer un rapport WCAG EARL",
+    "INTRO": "Si vous avez utilisé une autre application pour générer un rapport WCAG EARL qui entre dans le cadre de cette évaluation, vous pouvez l’importer et l’ajouter aux résultats de l’audit d’évaluation. Le fichier lui-même doit être conforme au format JSON-LD et composé d’objets au format EARL ; les tests d’évaluation doivent être liés aux WCAG.",
+    "LABEL_SELECT_FILE": "Sélectionner un fichier JSON-LD",
+    "IMPORT": {
+        "HEADING": "Résultat de l’importation {IMPORT_DATE, date, ::dd/MM/yyyy} par {AUTHOR}",
+        "TEST_PREFIX": "Test"
+    },
+    "ERROR": {
+    	"JSONLD_SYNTAX": "Le format de fichier n’est pas reconnu.",
+    	"NO_ASSERTIONS": "Le fichier ne contenait aucune affirmation reconnaissable.",
```

Comment:
I suggest to change "assertion/affirmation" by the French word "argument".
This use is more common. What do you think about "Le fichier ne contenait aucun argument reconnaissable." ?

Reply:
@remibetin : Unfortunaltely, this function doesn't work! It's not possible to get these messages "JSONLD_SYNTAX" and "NO_ASSERTIONS" (even if the files format is corrupted or if the file is empty).

The translation of the "NO_ASSERTIONS" value could be more simple by using present tense and "informations" instead of assertions : Le fichier ne contient aucune information rechargeable."